stdlib/LinearAlgebra/test/bidiag.jl

```diff
@@ -48,6 +48,10 @@ Random.seed!(1)
             # from matrix
             @test Bidiagonal(ubd, :U) == Bidiagonal(Matrix(ubd), :U) == ubd
             @test Bidiagonal(lbd, :L) == Bidiagonal(Matrix(lbd), :L) == lbd
+
+            z = rand(elty, 3, 3)
+            @test Bidiagonal(z, 'U') === Bidiagonal(z, :U)
```

Comment:
This test wont pass as `Bidiagonal` allocates 2 Vector to store the diag and upper/lower diag. 
Replace `===` with `==`?

Reply:
Done. Thanks for the catch.